src/Symfony/Component/Workflow/Registry.php

```diff
@@ -23,15 +23,19 @@ class Registry
     private $workflows = array();
 
     /**
-     * @param Workflow                 $workflow
-     * @param SupportStrategyInterface $supportStrategy
+     * @param Workflow                         $workflow
+     * @param WorkflowSupportStrategyInterface $supportStrategy
```

Comment:
the BC layer actually expects a `SupportStrategyInterface`? Using the new strategy interface should only go via `addWorkflow` IMHO.

Reply:
Agree